apis/placement/v1beta1/work_types.go

```diff
@@ -26,6 +26,20 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+// The following definitions are declared in the manager.go file.
+const (
+	// ManifestHashAnnotation is the annotation that indicates whether the spec of the object has been changed or not.
+	ManifestHashAnnotation = fleetPrefix + "spec-hash"
```

Comment:
These two fields are added in this PR but not referenced at all?

Reply:
They are in use by the work controller I assume?

Reply:
Will be used in the v1beta1 work controller in a separate PR.